woocommerce/woocommerce-ios
# Inbox view part 4: update empty state image

Part of #5954 


### Description


This PR added the image assets for the empty state of inbox notes `Inbox` view. I encountered some difficulties exporting the assets in Figma where the dark mode image has more padding than the light mode version. Eventually, I fixed it by copying the Figma design and then replacing the problematic mask layer in the dark mode with the light mode layer. 😅 

### Testing instructions


Prerequisite: having a site without inbox notes (some stores on older WC versions don't have them), or changing the code to always return the empty state

* Launch the app
* Go to the Menu tab
* Tap on "Inbox" --> after syncing, the empty state should look the same as design

### Screenshots


light | dark
-- | --
![Simulator Screen Shot - iPhone 11 - 2022-02-16 at 09 45 44](https://user-images.githubusercontent.com/1945542/154180484-610ec610-d93d-4cf7-a40f-9b7401832a8e.png) | ![Simulator Screen Shot - iPhone 11 - 2022-02-16 at 09 45 55](https://user-images.githubusercontent.com/1945542/154180491-ca28f93c-dce2-403b-aab9-6f324c5efcfa.png)


---
